stable/database/files/readinessprobe

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+function die() {
+  retval=$1
+  shift
+  echo "$@"
+  exit $retval
+}
+
+
+processes=`nuocmd show domain --process-format "{address} {start_id}" | grep \`hostname\` | awk '{print $2}'`
```

Comment:
Is this going to work if the user specifies `altAddr` for and engine? Most probably `grep` won't match anything and pod will never become ready.